_posts/2015-06-30-living-the-dream.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Living the dream
+category: post
+author_twitter: pietbrauer
+author_github: pietbrauer
+---
+
+I am happy to announce, that starting from July 1, 2015 my daily focus will shift from working full time at XING to being an Indie developer.
+
+My focus will be on building [Git2Go](http://git2go.com) and the open source software around it. While the AppStore tries to be a hostile place for independent developers, I hope there will be room for productivity software that enables iOS devices from being media consuming monsters to working machines for the go.
```

Comment:
.. that enable iOS devices to be working machines on the go, instead of being only media consuming monsters.